### PR TITLE
add a link to support when an unknown plugin error occurs

### PR DESCRIPTION
**Pull request by macqueen**
https://github.com/getsentry/sentry/pull/4392

@getsentry/plugins

### CODE REVIEW

**Comment by mattrobenolt** [comment]
This is going to be... awkward for on-premise won't it?

**Comment by macqueen** [comment]
@mattrobenolt they can use the forum. not going to merge this until this is live: https://github.com/getsentry/getsentry/pull/433

**Comment by mattrobenolt** [comment]
Ah, I thought this was directing them to emailing our support. Carry on. 👍

File: src/sentry_plugins/jira/static/jira/components/settings.jsx [hunk 1]
```diff
@@ -126,7 +126,7 @@ class Settings extends plugins.BasePlugin.DefaultSettings {
     if (this.state.state === FormState.ERROR && !this.state.fieldList) {
       return (
         <div className="alert alert-error m-b-1">
-          An unknown error occurred.
+          An unknown error occurred. Need help with this? <a href="https://sentry.io/support">Contact support</a>
```
Comment (mattrobenolt) [inline src/sentry_plugins/jira/static/jira/components/settings.jsx]: There will be a trailing slash here, otherwise it's a redirect.

**Comment by ehfeng** [comment]
I just merged.

**Pull request merged**